analyze_benchmark_data/benchmark_analysis.md

```diff
@@ -0,0 +1,174 @@
+
+``` r
+head(bm_loc_df)
+```
+
+    ##         file_id                                             text
+    ## 1 4641259a.html National Center for Genome Resources in Santa Fe
+    ## 2 4641259a.html                         University of California
+    ## 3 4641259a.html                                  Yale University
+    ## 4 4641259a.html                                               US
+    ## 5 4641259a.html                                       New Mexico
+    ## 6 4641259a.html                                      Connecticut
+    ##                 ner                   true_country true_un_region
+    ## 1      ORGANIZATION United States of America (USA)       Americas
+    ## 2      ORGANIZATION United States of America (USA)       Americas
+    ## 3      ORGANIZATION United States of America (USA)       Americas
+    ## 4           COUNTRY United States of America (USA)       Americas
+    ## 5 STATE_OR_PROVINCE United States of America (USA)       Americas
+    ## 6 STATE_OR_PROVINCE United States of America (USA)       Americas
+    ##   true_un_subregion
+    ## 1  Northern America
+    ## 2  Northern America
+    ## 3  Northern America
+    ## 4  Northern America
+    ## 5  Northern America
+    ## 6  Northern America
+
+``` r
+head(raw_loc_df)
+```
+
+    ##         file_id                                             text
+    ## 1 4641259a.html National Center for Genome Resources in Santa Fe
+    ## 2 4641259a.html                         University of California
+    ## 3 4641259a.html                                  Yale University
+    ## 4 4641259a.html                                               US
+    ## 5 4641259a.html                                       New Mexico
+    ## 6 4641259a.html                                      Connecticut
+    ##                 ner                    est_country est_un_region
+    ## 1      ORGANIZATION United States of America (USA)      Americas
+    ## 2      ORGANIZATION United States of America (USA)      Americas
+    ## 3      ORGANIZATION United States of America (USA)      Americas
+    ## 4           COUNTRY United States of America (USA)      Americas
+    ## 5 STATE_OR_PROVINCE United States of America (USA)      Americas
+    ## 6 STATE_OR_PROVINCE United States of America (USA)      Americas
+    ##   est_un_subregion
+    ## 1 Northern America
+    ## 2 Northern America
+    ## 3 Northern America
+    ## 4 Northern America
+    ## 5 Northern America
+    ## 6 Northern America
+
+Similar to before we will match columns baed on their names, in `raw_loc_df` it has `est_` columns and in `bm_loc_df` is has matching `true_` columns
+
+Now lets first look at the benchmark data ![](benchmark_analysis_files/figure-markdown_github/unnamed-chunk-9-1.png)![](benchmark_analysis_files/figure-markdown_github/unnamed-chunk-9-2.png)![](benchmark_analysis_files/figure-markdown_github/unnamed-chunk-9-3.png)
```

Comment:
Hmm. I think the difference would be more apparent if you had these as line plots that show magnitude changing over time. Might be a lil messier but easier to tease apart the differences.

Reply:
Good point! This is now updated to show line plots.